apps/spruce/src/components/Banners/TaskSchedulingWarningBanner.tsx

```diff
@@ -0,0 +1,20 @@
+import Banner from "@leafygreen-ui/banner";
+import { taskSchedulingLimitsDocumentationUrl } from "../../constants/externalResources";
+import { StyledLink } from "../styles";
```

Comment:
Same comment Sophie had about these auto-resolving as abs imports
```suggestion
import { taskSchedulingLimitsDocumentationUrl } from "constants/externalResources";
import { StyledLink } from "styles";
```